aea/test_tools/test_cases.py

```diff
@@ -785,6 +790,29 @@ def is_running(
 
         return missing_strings == []
 
+    def invoke(self, *args):
+        """Call the cli command."""
+        with cd(self._get_cwd()):
+            result = self.runner.invoke(
+                cli, args, standalone_mode=False, catch_exceptions=False
+            )
+        return result
+
+    def load_agent_config(self, agent_name: str) -> AgentConfig:
```

Comment:
Note, all other methods are class methods...

Reply:
Not sure why we need `invoke` when we have `run_cli_command`. This leads to further bloat at no benefit?

Reply:
I now see it's part of refactoring..let's refactor further :)